common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -293,6 +294,25 @@ def test_edit_container_on_container_page(self):
         container = self.go_to_nested_container_page()
         self.modify_display_name_and_verify(container)
 
+    def test_edit_raw_html(self):
+        """
+        Test the raw html editing functionality.
+        """
+        modified_content = "<p>modified content</p>"
+
+        #navigate to and open the component for editing
+        unit = self.go_to_unit_page()
+        container = unit.xblocks[1].go_to_container()
+        component = container.xblocks[1].children[0]
+        component.edit()
+
+        html_editor = HtmlComponentEditorView(self.browser, component.locator)
+        html_editor.set_content_and_save(modified_content, raw=True)
+
+        #quick-n-dirty regex to strip simple html tags
+        stripped_content = re.sub('<[^<]+?>', '', modified_content)
```

Comment:
http://stackoverflow.com/questions/753052/strip-html-from-strings-in-python

I know there's a builtin function for this in django, but that wouldn't be useful here would it?